eq-author-api/src/businessLogic/deleteLastPageRouting.js

```diff
@@ -0,0 +1,16 @@
+module.exports = ctx => {
+    if (
+        ctx.questionnaire &&
+        ctx.questionnaire.sections[0] &&
+        ctx.questionnaire.sections[0].pages[0]
+    ) {
+        const sections = ctx.questionnaire.sections.length;
+        const pages = ctx.questionnaire.sections[sections - 1].pages.length;
+
+        if (ctx.questionnaire.sections[sections - 1].pages[pages - 1] &&
+            ctx.questionnaire.sections[sections - 1].pages[pages - 1].routing) {
+          
+            delete ctx.questionnaire.sections[sections - 1].pages[pages - 1].routing;
```

Comment:
Nice work here. I'm suggesting changing the variable names to make it more clear that they contain the number of sections or pages, rather than the actual data.

```suggestion
        const nSections = ctx.questionnaire.sections.length;
        const nPages = ctx.questionnaire.sections[nSections - 1].pages.length;

        if (ctx.questionnaire.sections[nSections - 1].pages[nPages - 1] &&
            ctx.questionnaire.sections[nSections - 1].pages[nPages - 1].routing) {
          
            delete ctx.questionnaire.sections[nSections - 1].pages[nPages - 1].routing;
```